neqo-qpack/src/encoder.rs

```diff
@@ -81,7 +97,8 @@ impl QPackEncoder {
         let new_cap = std::cmp::min(self.max_table_size, cap);
         self.max_entries = new_cap / 32;
         // we also set our table to the max allowed.
-        self.change_capacity(new_cap)
+        self.change_capacity(new_cap);
```

Comment:
So the max capacity might not actually change? Maybe we don't want to call `change_capacity()` if it hasn't actually increased?